src/tasks.js

```diff
@@ -77,10 +77,10 @@ async function createTask(placeName) {
       .replace(/[^a-zA-Z]+/g, "-"); // Only keep [a-zA-Z]
   const name = client.taskPath(PROJECT, LOCATION, QUEUE, normalizedName);
   const task = {
-    name,
+    name, // Warning: Named Tasks cannot be re-created within a large timeframe (24h?)
```

Comment:
~1hr once the task has been removed from the queue. https://cloud.google.com/tasks/docs/quotas

Reply:
Thanks.
I think a different bug in my program led me to making this comment.
I improved the comment.